src/g3log/logmessage.hpp

```diff
@@ -45,8 +46,8 @@ namespace g3 {
       }
 
       /// use a different format string to get a different look on the time.
-      //  default look is Y/M/D H:M:S
-      std::string timestamp(const std::string& time_format = {internal::date_formatted + " " + internal::time_formatted}) const;
+      //  default look (Y/M/D H:M:S) is defined by default timestamp generator
+      std::string timestamp(const std::string& time_format = "") const;
```

Comment:
I can buy that we skip the default format argument and instead make it explicit. In that case I would remove the default argument altogether.